lib/kdump_utils.pm

```diff
@@ -90,7 +90,8 @@ sub activate_kdump {
     # activate kdump
     type_string "echo \"remove potential harmful nokogiri package boo#1047449\"\n";
     zypper_call('rm -y ruby2.1-rubygem-nokogiri', exitcode => [0, 104]);
-    script_run 'yast2 kdump', 0;
+    check_var('BACKEND', 'spvm') ? script_run 'yast2 --ncurses kdump', 0 : script_run 'yast2 kdump', 0;
```

Comment:
isn't ncurses by default on linux console ?

Reply:
connection to spvm is done via ssh, it will start yast gui without this option, see http://10.100.12.105/tests/1764#step/kdump_and_crash/33

Reply:
Migrated to new yast module and changed it to ` yast2_opts => '--ncurses'` which was recommended by qa-sle yast team.